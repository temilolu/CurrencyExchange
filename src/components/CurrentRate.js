import React from 'react';

const CurrentRate = ({ from, to, amount, result }) => {
	return (
		<div className='my-3'>
			<p className='font-weight-bold fsizes'>
				{amount} {from} = {result} {to}
			</p>
		</div>
	);
};

export default CurrentRate;
