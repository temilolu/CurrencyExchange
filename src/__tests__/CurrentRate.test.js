import React from 'react';
import CurrentRate from '../components/CurrentRate';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

afterEach(cleanup);

describe('<CurrentRate /> spec', () => {
	it('Should matches snapshot', () => {
		const { asFragment } = render(<CurrentRate from='USD' />);
		expect(asFragment()).toMatchSnapshot();
	});
});
