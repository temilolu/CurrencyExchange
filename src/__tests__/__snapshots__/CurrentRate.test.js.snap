// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrentRate /> spec Should matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="my-3"
  >
    <p
      class="font-weight-bold fsizes"
    >
       USD =  
    </p>
  </div>
</DocumentFragment>
`;
